classes/class-wplf-dynamic-values.php

```diff
@@ -0,0 +1,76 @@
+<?php
+if ( ! class_exists( 'WPLF_Polylang' ) ) {
+  class WPLF_Dynamic_Values {
+
+    public static $instance;
+    protected $regular_expression = "/%[^%%\n]+%/";
```

Comment:
I don't understand why this works. It doesn't work with double % signs, but it's copied directly from the Polylang class.